docs/contracts/lido.md

```diff
@@ -15,31 +16,58 @@ Lido is a liquid staking pool and the core contract that is responsible for:
 - applying fees and distributing rewards
 - passing buffered ether further to [StakingRouter](/contracts/staking-router) or [WithdrawalQueueERC721](/contracts/withdrawal-queue-erc721)
 
-Also, Lido is an [ERC-20](https://eips.ethereum.org/EIPS/eip-20) rebasing token, which represents staked ether, `stETH`. Tokens are minted upon deposit and burned when redeemed. `stETH` holder balances are updated daily with oracle reports. It also implements the [ERC-2612](https://eips.ethereum.org/EIPS/eip-2612) permit and [ERC-1271](https://eips.ethereum.org/EIPS/eip-1271) signature validation extensions.
+Also, Lido is an [ERC-20](https://eips.ethereum.org/EIPS/eip-20) rebasing token,
+which represents staked ether, `stETH`. Tokens are minted upon deposit and
```

Comment:
minted upon submit or submission (not deposit if refer to our established terms)